api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -543,6 +543,15 @@ message HttpConnectionManager {
   // emitted by the connection manager.
   repeated config.accesslog.v3.AccessLog access_log = 13;
 
+  // The interval to flush the above access logs. By default, the HCM will flush exactly one access log
+  // on stream close, when the HTTP request is complete. If this field is set, the HCM will flush access
+  // logs periodically at the specified interval. This is especially useful in the case of long-lived
+  // requests, such as CONNECT and Websockets. Final access logs can be detected via the
+  // `requestComplete()` method of `StreamInfo`.
```

Comment:
As a user, I'm interested in the access log format strings and how we can manifest it there clearly. Is there a formatter/filter already to distinguish?